### PR TITLE
Integrated file Explorer with file system 

### LINKED ISSUE #32: integrate file explorer to filesystem
The file explorer currently displays random documents and folders. We need to merge it to the existing filesystem.

### PR DESCRIPTION
**Pull request by ankushbhardwxj**
fixes #32 
Here we can see that files from the filesystem are fetched and brought in the left pane. The left pane will be a tree of files like windows. This is the initial implementation. 
Capture

### CODE REVIEW

**Comment by ankushbhardwxj** [comment]
@RAJAGOPALAN-GANGADHARAN please merge this 👍 
The terminal is almost complete. I've started working on this file explorer. I will get this done all by myself 👍 🥇 Let's see how dope I can make this OS. lol

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
Haha sure, did this as some stupid hobby but had a lot of fun. Hope you have lot of fun too.

File: src/defaultapps/explorer/explorer.js [hunk 1]
```diff
@@ -1,119 +1,70 @@
...
-            </div>
+            
         )
+
     }
-}
\ No newline at end of file
+}
+
+export default Explorer
```
Comment (RAJAGOPALAN-GANGADHARAN) [inline src/defaultapps/explorer/explorer.js]: New line please.

Comment (ankushbhardwxj) [reply src/defaultapps/explorer/explorer.js]: Done

**Comment by ankushbhardwxj** [comment]
@RAJAGOPALAN-GANGADHARAN yeah, I had fun + I learnt a lot of stuff.

**Comment by ankushbhardwxj** [comment]
@RAJAGOPALAN-GANGADHARAN This can be merged now 👍

**Pull request merged**